activelearning/sampler/sampler.py

```diff
@@ -157,3 +157,25 @@ def fit(self):
     def get_samples(self, n_samples, candidate_set=None):
         batch, times = self.sampler.sample_batch(n_forward=n_samples, train=False)
         return (batch.get_terminating_states(proxy=True), None)
+
+
+class RandomGFlowNetSampler(Sampler):
+    def __init__(self, acquisition, conf, device, float_precision):
+        super().__init__(acquisition, device, float_precision)
+        import hydra
+
+        env_maker = hydra.utils.instantiate(
+            conf.env,
+            device=device,
+            float_precision=float_precision,
+            _partial_=True,
+        )
+        self.env = env_maker()
```

Comment:
There is no need to make an environment maker, to then instantiate one environment.
```suggestion
        self.env = hydra.utils.instantiate(
            conf.env,
            device=device,
            float_precision=float_precision,
        )
```

Reply:
Also, why do you need to store the env as an attribute of the class (`self`)?

Reply:
to use it for inference later